pkg/operation/botanist/health_check.go

```diff
@@ -306,12 +306,12 @@ func (b *Botanist) checkAPIServerAvailability(condition *gardenv1beta1.Condition
 		} else {
 			body = string(bodyRaw)
 		}
-		message := fmt.Sprintf("Shoot API server health check returned a non ok status code %d. %s (%s)", statusCode, responseDurationText, body)
+		message := fmt.Sprintf("Shoot API server /healthz endpoint endpoint check returned a non ok status code %d. %s (%s)", statusCode, responseDurationText, body)
 		return helper.UpdatedCondition(condition, corev1.ConditionFalse, "ResponseCodeError", message)
 	}
 
-	message := fmt.Sprintf("Shoot API server responded with success status code. %s", responseDurationText)
-	return helper.UpdatedCondition(condition, corev1.ConditionTrue, "ResponseCodeOK", message)
+	message := fmt.Sprintf("Shoot API server /healthz endpoint responded with success status code. %s", responseDurationText)
+	return helper.UpdatedCondition(condition, corev1.ConditionTrue, "ResponseCodeOK from API server /healthz endpoint", message)
```

Comment:
`Request to  API server /healthz endpoint failed` (condition reason) should be one word: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#typical-status-properties

How about `HealthzRequestSucceeded`?